### PR TITLE
yazi: update to 0.2.0

### PR DESCRIPTION
**Pull request by eatradish**
Topic Description
-----------------

- yazi: update to 0.2.0

Package(s) Affected
-------------------

- yazi: 0.2.0

Security Update?
----------------

No

Build Order
-----------

```
#buildit yazi
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`

**Secondary Architectures**

- [ ] Loongson 3 `loongson3`
- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Comment by eatradish** [comment]
@aosc-buildit-bot build

**Comment by eatradish** [comment]
LGTM on AMD64:

图片

File: app-utils/yazi/autobuild/prepare [hunk 1]
```diff
@@ -0,0 +1,3 @@
+abinfo "set var to general completions ..."
+export YAZI_GEN_COMPLETIONS=1
+
```
Comment (MingcongBai) [inline app-utils/yazi/autobuild/prepare]: Extraneous new line.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Comment by eatradish** [comment]
@aosc-buildit-bot build

**Pull request merged**